### PR TITLE
Broken formatting

### LINKED ISSUE #164: CMP formatting displayed differently on various websites
The CMP UI is displayed differently on some of the tested websites, e.g. buttons not displayed correctly, different font sizes, changed alignment of the texts, changed or disappeared bullet points - see attached screenshots. I am also attaching the configuration file. The source code was taken from the master branch (last Friday 31/08).
**Tested sites**:
www.orbion.cz
www.ahaonline.cz
www.maminka.cz
**Browsers**: Chrome and Safari
**Note 1**: We had to take the CMP down from those sites, but you can still display it online on Chrome with a DTM Launch switch (stageing ON). 
**Note 2**: The variable PageviewCMP was added on our side for GA tracking.

ahaonline - buttons
maminka - alignment
maminka - alignment 2
orbion - font and bullet points

### PR DESCRIPTION
**Pull request by rusel1989**
closes #164

### CODE REVIEW

File: src/components/popup/intro/intro.less [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 div.intro {
 	align-items: center;
-	padding: 0 4em;
+	/*padding: 0 4em;*/
```
Comment (shackijj) [inline src/components/popup/intro/intro.less]: This should probably be deleted

**Comment by shackijj** [comment]
Was the problem discussed in #164 specific for a certain website? I'm asking because I can't reproduce it locally.

Overall, it seems that these changes could help in some cases when a websites' styles conflict with CMP's styles.

**Comment by lukascpex** [comment]
Hi,
regarding your question about the #164 - yes, the concerned sites were:
 [www.orbion.cz](http://www.orbion.cz)
[www.ahaonline.cz](http://www.ahaonline.cz)
[www.maminka.cz](http://www.maminka.cz)

File: src/components/popup/intro/intro.less (hunk 1 above)
Comment (rusel1989) [reply src/components/popup/intro/intro.less]: The version you've made this comment on seems outdated. I can't see the comment in the files changed view...

**Comment by rusel1989** [comment]
@samtingleff can we merge this please ?

**Pull request merged**